apps/communications/contacts/js/views/details.js

```diff
@@ -435,68 +425,18 @@ contacts.Details = (function() {
   };
 
   var renderSocial = function cd_renderSocial(contact) {
```

Comment:
Do we need this method? This is called if FB is enabled, and this scenario will not be available in the future.